CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -39,9 +39,12 @@ SOFTWARE.
 
 #include "th_map.h"
 
-#if !SDL_VERSION_ATLEAST(2, 0, 10)
+#if SDL_VERSION_ATLEAST(2, 0, 10)
```

Comment:
Do we still see improvement in older SDL?

Reply:
Yes, the same improvement. This change draws the floating dollar icon and text into it's native size before scaling it up / down to the screen. I.e. supposing its sprite size is 80x50, it draws the text and icon into a texture at that size and then scales the entire thing. This is the same idea as how https://github.com/CorsixTH/CorsixTH/pull/1934 preserved text quality for full screen but applied on the small scale of an individual sprite rather than the entire screen.